.env

```diff
@@ -0,0 +1,2 @@
+export SENTRY_URL="https://d24382d743314dadadb80d1eabdb139e@sentry.pennlabs.org/13"
```

Comment:
Don't commit this file! Add it to the `.gitignore` and ensure that it is removed before merging.